src/audio/detect_test.c

```diff
@@ -251,6 +254,13 @@ static struct comp_dev *test_keyword_new(struct sof_ipc_comp *comp)
 		}
 	}
 
+	ret = alloc_mem_load(cd, INITIAL_MODEL_DATA_SIZE);
+	if (ret < 0) {
+		trace_keyword_error("test_keyword_new() "
+				"error: model data initial failed");
```

Comment:
Looks like indent is off. Didn't Checkpatch  complain?

Reply:
Thanks, it's strange Checkpatch didn't complain here to me. Correcting...